dataflow/flex-templates/streaming_beam/streaming_beam.py

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/env python
+#
+# Copyright 2020 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""An Apache Beam streaming pipeline example.
+
+It reads JSON encoded messages from Pub/Sub, transforms the message data and
+writes the results to BigQuery.
+"""
+
+from __future__ import absolute_import
+
+import argparse
+import json
+import logging
+import time
+
+import apache_beam as beam
+from apache_beam.options.pipeline_options import PipelineOptions
```

Comment:
This gets flagged as a lint error ("invalid import error") in cider. I can still make that change if you think that is the right styling.

Reply:
mmm interesting, even if you separate the `import ...` from the `from .. import ..` with a blank line? I wander what that "invalid import error" means, it sounds like it's not finding the library or something?

Reply:
Yes, i get this lint error even if there is a blank line. Also, this is flagged by cider lint tool. I guess the reason is because it expects all the import/from lines to be lexicographically sorted. I am not sure whether we follow the same styling in this project.

Reply:
Let's leave it so there are no lint errors. Thanks for checking.